conferences.json

```diff
@@ -90,5 +90,25 @@
             "wheelchair accessible": "yes",
             "childcare": "yes"
         }
+    },
+        {
+        "name": "FOSDEM 2020",
+        "date": "2020-02-02T00:00:00+01:00",
```

Comment:
@XAMPPRocky I think there is something fishy going on, as on my desktop browser (FF on Linux) this is shown as `Saturday, 1st of February 2020` while on my mobile (FF on Android) this is correctly displayed on Sunday.

Reply:
Here's the line of code that does this. I suspect it's a bug on your end or in `moment.js`.
https://github.com/XAMPPRocky/timetill.rs/blob/cca365f39e2c1136af799b75e71c0dbf05b01c1d/src/main.js#L8